tests/integration/test_tiler.py

```diff
@@ -123,7 +123,7 @@ def it_locates_tiles_on_the_slide(
         slide = Slide(fixture_slide, "")
         random_tiles_extractor = RandomTiler(
             tile_size=tile_size,
-            n_tiles=2,
+            n_tiles=100,
```

Comment:
did you consider the performance during testrunner increasing the n tiles of 2 order of magnitude?

Reply:
I did consider it, but the point is that with only 2 tiles we cannot really see how the tiler is sampling the tiles, if you compare the old and new expectations you will see that only now this test is really meaninfgul